Add weighted modifier to modifiers doc (#13657)
DataDog/documentation: content/en/metrics/type_modifiers.md

{{% /tab %}}
{{< /tabs >}}

### The `weighted()` modifier

> High tag churn is seen for tags like `pod name` or `container_name` especially when creating queries for cost management, capacity planning or autoscaling for containerized applications. To ensure mathematical accuracy of queries on gauges regardless of tag churn, there exists a `.weighted()` in-application modifier that guarantees metrics' values are properly weighted based on the lifespan of these frequently churning tags. 

Reviewer: ```suggestion
Tags like `pod name` or `container_name` cause high tag churn, especially when creating queries for cost management, capacity planning or autoscaling for containerized applications. To ensure mathematical accuracy of queries on gauges regardless of tag churn, there exists a `.weighted()` in-application modifier. The `.weighted()` modifier ensures that Datadog properly weights metric values based on the lifespan of these frequently churning tags. 
```

The `.weighted()` modifier is automatically appended to queries on gauges only if both of the following conditions are met:

> - The gauge metric is submitted regularly, such that there is no interpolation over gaps

Reviewer: ```suggestion
- The gauge metric is submitted regularly, such that there is no interpolation over gaps.
```

- The submission interval is correctly defined and set. 

> Submission intervals are a part of a metric's metadata that can be set at intake by either the Datadog Agent or our integrations; however, they can be modified on the [Metrics Summary page][4].

Reviewer: ```suggestion

Either the Datadog Agent or an integration sets the submission interval for a metric at time of intake. Modify submission intervals on the [Metrics Summary page][4].
```
